pkg/util/utils.go

```diff
@@ -609,3 +609,46 @@ func Tmpdir() string {
 
 	return tmpdir
 }
+
+// ValidateSysctls validates a list of sysctl and returns it.
+func ValidateSysctls(strSlice []string) (map[string]string, error) {
```

Comment:
Looks like a candidate for a `c/common/pkg/sysctl`.

Reply:
WIll open a different PR for this, but I would prefer to get this merged and fixed later.

Reply:
Completely agree :+1:

Reply:
https://github.com/containers/common/pull/97